### ISSUE TITLE
Newest 1.0.10 doesn't work via NPM package

### ISSUE THREAD

**Issue by laurilarjo** [ToeFungi/exchange-rates-as-promised#20]
Hi,

I just updated your package from v1.0.4 -> v1.0.10, but now when I try to create a new ExchangeRate object, it throws the following exception:
`TypeError: exchange_rates_as_promised_1.ExchangeRate is not a constructor`

I'm using it like this: 
```
import { ExchangeRate, Currencies } from 'exchange-rates-as-promised';
const exchangeRate = new ExchangeRate();
...
```

Same code worked on 1.0.4. Based on your docs or commit history, I didn't see any changes on how it should be used, so I suspect the issue is with the NPM package you create.

**Comment by ToeFungi** [comment]
Hi @laurilarjo 

Thanks for bringing this to my attention. I'll investigate and try to get a fix released shortly.

Not sure why, but GitHub doesn't seem to send notifications when an issue is raised.

**Comment by ToeFungi** [comment]
It would appear that it's some issue with the webpack build. I've published v1.0.12 which should work.

**Comment by laurilarjo** [comment]
Thank you! I've verified the 1.0.12 works correctly. I'll close this issue. 
Also noticed the 1.0.12 still has the vulnerable axios, but it seems you've just updated that with latest commit 👍
